src/server/JasmineGraphServer.cpp

```diff
@@ -889,21 +890,25 @@ std::vector<JasmineGraphServer::worker> JasmineGraphServer::workers(size_t npart
 void JasmineGraphServer::uploadGraphLocally(int graphID, const string graphType,
                                             vector<std::map<int, string>> fullFileList, std::string masterIP) {
     server_logger.info("Uploading the graph locally..");
-    std::map<int, string> partitionFileMap = fullFileList[0];
-    std::map<int, string> centralStoreFileMap = fullFileList[1];
-    std::map<int, string> centralStoreDuplFileMap = fullFileList[2];
-    std::map<int, string> compositeCentralStoreFileMap = fullFileList[5];
-    std::map<int, string> attributeFileMap;
-    std::map<int, string> centralStoreAttributeFileMap;
+    std::unordered_map<int, string> partitionFileMap = 
+        std::unordered_map<int, string>(fullFileList[0].begin(), fullFileList[0].end());
+    std::unordered_map<int, string> centralStoreFileMap = 
+        std::unordered_map<int, string>(fullFileList[1].begin(), fullFileList[1].end());
+    std::unordered_map<int, string> centralStoreDuplFileMap = 
+        std::unordered_map<int, string>(fullFileList[2].begin(), fullFileList[2].end());
+    std::unordered_map<int, string> compositeCentralStoreFileMap = 
+        std::unordered_map<int, string>(fullFileList[5].begin(), fullFileList[5].end());
+    std::unordered_map<int, string> attributeFileMap;
+    std::unordered_map<int, string> centralStoreAttributeFileMap;
     if (masterHost.empty()) {
         masterHost = Utils::getJasmineGraphProperty("org.jasminegraph.server.host");
     }
     int total_threads = partitionFileMap.size() + centralStoreFileMap.size() + centralStoreDuplFileMap.size() +
                         compositeCentralStoreFileMap.size();
     if (graphType == Conts::GRAPH_WITH_ATTRIBUTES) {
-        attributeFileMap = fullFileList[3];
+        attributeFileMap = std::unordered_map<int, string>(fullFileList[3].begin(), fullFileList[3].end());
```

Comment:
Rather than having an index here can we have a constant with a meaningful name for the index value and use the constant here?

Reply:
Ok